src/bin/scheduler/scheduling/scheduler.rs

```diff
@@ -43,7 +45,7 @@ pub async fn run_plans_and_cleanup(global_config: &GlobalConfig, plans: &[Plan])
     info!("Received termination signal while scheduling, waiting for plans to terminate");
     while let Some(outcome) = join_set.join_next().await {
         if let Err(error) = outcome {
-            error!("{error:?}");
+            log_and_return_error(error);
```

Comment:
What is the difference to just logging the error?

Reply:
In case we decide to log errors differently, this would be taken into account here.